Battery.elm

```diff
@@ -1,35 +1,48 @@
 module Battery exposing (Battery, recommend)
 
-import Usage exposing (Usage)
+import List exposing (head, map, repeat, sortBy, sum)
 import Load exposing (getAmps)
-import List exposing (sum, map, repeat, sortBy, head)
+import Usage exposing (Usage)
+
+
 -- MODEL
 
+
 type alias Battery =
     { name : String
     , maxDischargePercentage : Int
     , ah : Int
     , price : Float
     }
 
-liFePO4 = Battery "LiFePO4" 80
 
 batteries : List Battery
 batteries =
+    let
+        liFePO4 =
+            Battery "LiFePO4" 80
+    in
     [ liFePO4 40 215.04
     , liFePO4 60 309.12
     , liFePO4 90 483.84
     ]
 
-recommend : Int -> List Usage -> Maybe ( List Battery )
+
+recommend : Int -> List Usage -> Maybe (List Battery)
```

Comment:
why do you return Maybe List? is there any good reason for that?

Reply:
No no. I wanted to return Battery Pack. Just `List Battery` but the head option is `Maybe`

Reply:
yeah because what is the head of empty list.... Anyway you can try non empty list instead http://package.elm-lang.org/packages/mgold/elm-nonempty-list/latest

Reply:
Oh yes, that makes sense, that's why I added there the Maybe (and also because the compiler cried) Thanks for the nonempty list. As this will always return something